### PR TITLE
Fix tutor typo `favourite` to `favorite`

### PR DESCRIPTION
**Pull request by s0LA1337**
Fixed the mismatch of the word `favourite` and `favorite` in the tutor.
So i fixed that to be synchronous now.

### CODE REVIEW

File: runtime/tutor [hunk 1]
```diff
@@ -519,7 +519,7 @@ _________________________________________________________________
     like 'oranges'.
  6. Type , to remove the second cursor.
 
- --> I like to eat apples since my favorite fruit is apples.
+ --> I like to eat apples since my favourite fruit is apples.
      I like to eat oranges since my favourite fruit is oranges.
```
Comment (the-mikedavis) [inline runtime/tutor:523]: actually I think the typo is the one with the `u` - there's another `favorite` down on L833

Comment (David-Else) [reply runtime/tutor:523]: https://www.grammarly.com/blog/favorite-favourite/

`favorite` is American English, it is `favourite` everywhere else.

Comment (the-mikedavis) [reply runtime/tutor:523]: It seems like we use American English elsewhere in the tutor, but the other "favorite" is the only example I found of that. I don't have strong feelings either way between American vs. British

Comment (s0LA1337) [reply runtime/tutor:523]: so i assume i will fix it up to use the AE version.

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks!

**Pull request merged**